cmd/storage-rest-client.go

```diff
@@ -669,7 +668,6 @@ func newStorageRESTClient(endpoint Endpoint) *storageRESTClient {
 
 	trFn := newInternodeHTTPTransport(tlsConfig, rest.DefaultRESTTimeout)
 	restClient := rest.NewClient(serverURL, trFn, newAuthToken)
-	restClient.HealthCheckInterval = 500 * time.Millisecond
```

Comment:
Can't we make it something small.? 10 or 50ms or something similar.

Reply:
Isn't 200ms enough?